onedal/covariance/incremental_covariance.py

```diff
@@ -136,7 +148,12 @@ def finalize_fit(self, queue=None):
             self._partial_result,
         )
         if daal_check_version((2024, "P", 1)) or (not self.bias):
-            self.covariance_ = from_table(result.cov_matrix)
+            self.covariance_ = from_table(
```

Comment:
When running sklearnex example incremental_covariance_spmd.py leads to AttributeError: 'IncrementalEmpiricalCovariance' object has no attribute '_input_sua_iface'